scripts/svm/acrossPlusJupiter.ts

```diff
@@ -0,0 +1,348 @@
+          state: AddressLookupTableAccount.deserialize(accountInfo.data),
+        });
+        acc.push(addressLookupTableAccount);
+      }
+
+      return acc;
+    }, []);
+  };
+
+  const addressLookupTableAccounts = await getAddressLookupTableAccounts(instructions.addressLookupTableAddresses);
+
+  // Helper to deserialize instruction and check if it would fit in inner CPI limit.
+  const deserializeInstruction = (instruction: any) => {
+    const transactionInstruction = new TransactionInstruction({
+      programId: new PublicKey(instruction.programId),
+      keys: instruction.accounts.map((key: any) => ({
+        pubkey: new PublicKey(key.pubkey),
+        isSigner: key.isSigner,
+        isWritable: key.isWritable,
+      })),
+      data: Buffer.from(instruction.data, "base64"),
+    });
+    const innerCpiLimit = 1280;
+    const innerCpiSize = transactionInstruction.keys.length * 34 + transactionInstruction.data.length;
+    if (innerCpiSize > innerCpiLimit) {
+      throw new Error(
+        `Instruction too large for inner CPI: ${innerCpiSize} > ${innerCpiLimit}, try lowering maxAccounts`
+      );
+    }
+    return transactionInstruction;
+  };
+
+  // Ignore Jupiter setup instructions as we need to create ATA both for the recipient and the handler signer.
+  const createHandlerATAInstruction = createAssociatedTokenAccountIdempotentInstruction(
+    handlerSigner,
+    handlerOutputTA,
+    handlerSigner,
+    outputMint,
+    outputTokenProgram
+  );
+  const createRecipientATAInstruction = createAssociatedTokenAccountIdempotentInstruction(
+    handlerSigner,
+    recipientOutputTA,
+    recipient,
+    outputMint,
+    outputTokenProgram
+  );
+
+  // Construct ix to transfer minimum output tokens from handler to the recipient ATA. Note that all remaining tokens
+  // can be stolen by anyone. This could be improved by creating a sweeper program that reads actual handler ATA balance
+  // and transfers all of them to the recipient ATA.
+  const outputDecimals = (await getMint(provider.connection, outputMint, undefined, outputTokenProgram)).decimals;
+  const transferInstruction = createTransferCheckedInstruction(
+    handlerOutputTA,
+    outputMint,
+    recipientOutputTA,
+    handlerSigner,
+    quoteResponse.otherAmountThreshold,
+    outputDecimals,
+    undefined,
+    outputTokenProgram
+  );
+
+  // Encode all instructions with handler PDA as the payer for ATA initialization.
+  const multicallHandlerCoder = new MulticallHandlerCoder(
+    [
+      createHandlerATAInstruction,
+      deserializeInstruction(instructions.swapInstruction),
+      createRecipientATAInstruction,
+      transferInstruction,
+    ],
+    handlerSigner
+  );
+  const handlerMessage = multicallHandlerCoder.encode();
+  const message = new AcrossPlusMessageCoder({
+    handler: handlerProgram.programId,
+    readOnlyLen: multicallHandlerCoder.readOnlyLen,
+    valueAmount: new BN(valueAmount), // Must exactly cover ATA creation.
```

Comment:
Question: what happens if handlerATA or recipientATA already exist? Would this error?